build.gradle

```diff
@@ -117,7 +117,7 @@ subprojects { project ->
         maxParallelForks = Runtime.runtime.availableProcessors()
     }
 
-    build.dependsOn ':detekt'
+   // build.dependsOn ':detekt'
```

Comment:
why?

Reply:
It throws error on files in "generated" directory - I must finally fix it in config

Reply:
We only run detekt on clean builds. We normally use `assemble` and `test` rather than `build` to make the lib!